code/modules/reagents/chemistry/reagents/cit_reagents.dm

```diff
@@ -0,0 +1,274 @@
+			var/mob/living/carbon/human/H = M
+			if(prob(50)) //Less spam
+				to_chat(H, "<span class='love'>Your libido is going haywire!</span>")
+				H.mob_climax(forced_climax=TRUE)
+		if(M.min_arousal < 50)
+			M.min_arousal += 1
+		if(M.min_arousal < M.max_arousal)
+			M.min_arousal += 1
+		M.adjustArousalLoss(2)
+	..()
+
+/datum/reagent/drug/anaphrodisiac
+	name = "Camphor"
+	id = "anaphro"
+	description = "Naturally found in some species of evergreen trees, camphor is a waxy substance. When injested by most animals, it acts as an anaphrodisiac\
+					, reducing libido and calming them. Non-habit forming and not addictive."
+	taste_description = "dull bitterness"
+	taste_mult = 2
+	color = "#D9D9D9"//rgb(217, 217, 217)
+	reagent_state = SOLID
+
+/datum/reagent/drug/anaphrodisiac/on_mob_life(mob/living/M)
+	if(M && M.canbearoused && prob(33))
+		M.adjustArousalLoss(-2)
+	..()
+
+/datum/reagent/drug/anaphrodisiacplus
+	name = "Hexacamphor"
+	id = "anaphro+"
+	description = "Chemically condensed camphor. Causes an extreme reduction in libido and a permanent one if overdosed. Non-addictive."
+	taste_description = "tranquil celibacy"
+	color = "#D9D9D9"//rgb(217, 217, 217)
+	reagent_state = SOLID
+	overdose_threshold = 20
+
+/datum/reagent/drug/anaphrodisiacplus/on_mob_life(mob/living/M)
+	if(M && M.canbearoused && prob(33))
+		M.adjustArousalLoss(-4)
+	..()
+
+/datum/reagent/drug/anaphrodisiacplus/overdose_process(mob/living/M)
+	if(M && M.canbearoused && prob(33))
+		if(M.min_arousal > 0)
+			M.min_arousal -= 1
+		if(M.min_arousal > 50)
+			M.min_arousal -= 1
+		M.adjustArousalLoss(-2)
+	..()
+
+//recipes
+/datum/chemical_reaction/aphro
+	name = "crocin"
+	id = "aphro"
+	results = list("aphro" = 6)
+	required_reagents = list("carbon" = 2, "hydrogen" = 2, "oxygen" = 2, "water" = 1)
+	required_temp = 400
+	mix_message = "The mixture boils off a pink vapor..."//The water boils off, leaving the crocin
+
+/datum/chemical_reaction/aphroplus
+	name = "hexacrocin"
+	id = "aphro+"
+	results = list("aphro+" = 1)
+	required_reagents = list("aphro" = 6, "phenol" = 1)
+	required_temp = 400
+	mix_message = "The mixture rapidly condenses and darkens in color..."
+
+/datum/chemical_reaction/anaphro
+	name = "camphor"
+	id = "anaphro"
+	results = list("anaphro" = 6)
+	required_reagents = list("carbon" = 2, "hydrogen" = 2, "oxygen" = 2, "sulfur" = 1)
+	required_temp = 400
+	mix_message = "The mixture boils off a yellow, smelly vapor..."//Sulfur burns off, leaving the camphor
+
+/datum/chemical_reaction/anaphroplus
+	name = "pentacamphor"
+	id = "anaphro+"
+	results = list("anaphro+" = 1)
+	required_reagents = list("anaphro" = 5, "acetone" = 1)
+	required_temp = 300
+	mix_message = "The mixture thickens and heats up slighty..."
+
+//=========Drinks and Stuff!============//
+
+/datum/reagent/consumable/ethanol/sake
+	name = "Sake"
+	id = "sake"
+	description = "A sweet rice wine of questionable legality and extreme potency."
```

Comment:
Hahaha okay though but when are y'all gonna port this weeb shit

Reply:
ooh! i coded this!